src/integrationTest/groovy/wooga/gradle/fastlane/tasks/SighRenewIntegrationSpec.groovy

```diff
@@ -86,41 +83,33 @@ class SighRenewIntegrationSpec extends AbstractFastlaneTaskIntegrationSpec {
         "ignoreProfilesWithDifferentName" | "ignoreProfilesWithDifferentName.set" | false                      | "Boolean"      || "--ignore_profiles_with_different_name false"
         "ignoreProfilesWithDifferentName" | _                                     | _                          | "Boolean"      || "--ignore_profiles_with_different_name false"
         "fileName"                        | "fileName.set"                        | "test2.mobileprovisioning" | "String"       || "--filename test2.mobileprovisioning"
-        "destinationDir"                  | "destinationDir.set"                  | "/some/path"               | "File"         || "--output_path /some/path"
+        "destinationDir"                  | "destinationDir.set"                  | "some/path"                | "File"         || "--output_path some/path"
         "additionalArguments"             | "setAdditionalArguments"              | ["--verbose", "--foo bar"] | "List<String>" || "--verbose --foo bar"
-        "apiKeyPath"                      | "apiKeyPath.set"                      | "/path/to/key.json"        | "File"         || "--api-key-path /path/to/key.json"
+        "apiKeyPath"                      | "apiKeyPath.set"                      | "path/to/key.json"         | "File"         || "--api-key-path path/to/key.json"
         value = wrapValueBasedOnType(rawValue, type)
         valueMessage = (rawValue != _) ? "with value ${value}" : "without value"
     }
 
     @Unroll("property #property #valueMessage sets environment #expectedEnvironmentPair")
     def "constructs process environment"() {
-        given: "a task to read the build arguments"
-        buildFile << """
-            task("readValue") {
-                doLast {
-                    println("arguments: " + ${getTestTaskName()}.environment.get().collect {k,v -> k + '=' + v}.join("\\n"))
-                }
-            }
-        """.stripIndent()
-
-        and: "a set property"
+        given: "a set property"
         if (method != _) {
             buildFile << """
             ${getTestTaskName()}.${method}($value)
             """.stripIndent()
         }
 
         when:
-        def result = runTasksSuccessfully("readValue")
+        def result = runTasksSuccessfully(testTaskName)
 
         then:
-        outputContains(result, expectedEnvironmentPair)
+        def environmentsLines = result.standardOutput.split(/\[ENVIRONMENT\]\:${System.getProperty("line.separator")}/).last().readLines()
```

Comment:
You can use the `BatchmodeWrapper.containsEnvironment` method here instead. Give it a try!

Reply:
Ok I added this now.